docs/wiki/Deploying-Enterprise-Scale-HubAndSpoke.md

```diff
@@ -56,7 +61,7 @@ Depending on your requirements, you may choose to deploy additional network infr
 * VPN and ExpressRoute Gateways
   * If you choose to deploy either or both of these gateways, you will have the option to select the subnet to be dedicated for these resources, if you decide to deploy them as regional or zone-redundant gateways, as well as choose the right SKU based on your requirements
 * Azure Firewall
-  * If you choose to deploy Azure Firewall, you will have the option to select the subnet, select to deploy the Firewall as regional or zone redundant as well as indicate if you want to enable DNS Proxy in Azure Firewall
+  * If you choose to deploy Azure Firewall, you will have the option to select the subnet, select to deploy the Firewall as regional or zone redundant, select the Firewall SKU as well as indicate if you want to enable DNS Proxy in Azure Firewall
```

Comment:
maybe be more prescriptive and mention Azure Firewall Premium Features

...You may choose to enable Azure Firewall Premium...